js/components/addressBookList/addressBookList_controller.js

```diff
@@ -7,6 +7,14 @@ angular.module('contactsApp')
 	AddressBookService.getAll().then(function(addressBooks) {
 		ctrl.addressBooks = addressBooks;
 		ctrl.loading = false;
+		if(ctrl.addressBooks.length === 0) {
+			AddressBookService.create(t('contacts', 'Contacts')).then(function() {
```

Comment:
also, I would expect the service to return the newly created address book. Then you don't have to query it below.